posts/Yoann/2020-12-11-hibernate-search-6-0-0-Final.adoc

```diff
@@ -0,0 +1,295 @@
+
+The Elasticsearch backend now works with Elasticsearch 5.6, 6.8 or 7.10.
+Support for older versions of Elasticsearch was dropped.
+
+[[api-refresh]]
+=== API overhaul
+
+This is perhaps the most important aspect of Hibernate Search 6: most APIs have changed.
+We didn't take this decision lightly, and we don't intend to do this again anytime soon:
+at this point, breaking APIs was necessary for Hibernate Search to keep improving.
+
+The primary goal of this overhaul was to abstract away Lucene APIs,
+so that the Elasticsearch backend can be become a first-class citizen,
+implemented without any hacks or unnecessary dependencies.
+
+It goes without saying, but the Lucene backend will remain as important as ever.
+In fact, abstracting away Lucene APIs is good for the Lucene backend, too:
+it will give the Hibernate Search project the ability
+to upgrade Lucene as soon as new versions are released.
+
+Another benefit of the overhaul: since APIs had to change anyway, we took that opportunity to implement
+long-overdue improvements that also required API changes.
+See the following sections to know more about these improvements.
+
+=== Safer, more concise, backend-agnostic Search DSL
+
+The link:{hsearch-doc-url-prefix}#search-dsl[Search DSL] is brand new, with several improvements:
+
+Safer::
+* Search hits link:{hsearch-doc-url-prefix}#search-dsl-query-fetching-results[now have a type],
+even when projecting thanks to the brand-new link:{hsearch-doc-url-prefix}#search-dsl-projection[projection DSL].
+* The implementation for numeric predicates (e.g. `int`-based, `long`-based, `double`-based)
+is now determined from the type of the targeted field, instead of the type of arguments.
+This is safer when passing arguments with the wrong type:
+where Hibernate Search 5 would have generated a predicate that doesn't match anything,
+Hibernate Search 6 will detect the type mismatch and will throw an exception.
+
+More concise::
+* Ability to link:{hsearch-doc-url-prefix}#search-dsl-predicate[use lambdas]
+for more concise query definition,
+link:{hsearch-doc-url-prefix}#search-dsl-predicate-boolean-lambda[even when queries are complex].
+* link:{hsearch-doc-url-prefix}search-dsl-query-targeting-multiple[Simpler syntax for predicates when targeting multiple types] in a single query:
+instantiating multiple `QueryBuilders` is no longer needed,
+Hibernate Search takes into account that multiple types are targeted
+and automatically checks that targeted fields are compatible across all targeted indexes.
+
+More consistent::
+* The API types consistently use the `Search` prefix: no more mixing `FullText` with `Search` or simply no prefix.
+* The `SearchQuery` type (previously `FullTextQuery`)
+now link:{hsearch-doc-url-prefix}#search-dsl-query-fetching-results[defines its own methods]
+instead of extending JPA's `TypeQuery`,
+highlighting the fact that it targets an index and not a database, and making it a friendlier API.
+It is still possible to
```

Comment:
I would add a conjunction e.g.: `Even if it is still possible ...`, to make the text more readable.